### PR TITLE
Select by district implemented

### PR DESCRIPTION
**Pull request by rkaufman13**
Districts can now be selected once a user selects a state to create a new order.

The district info is hardcoded in states.js. I generated it using a quick & dirty script, happy to push that as well if it would be useful.

### CODE REVIEW

**Comment by thadk** [comment]
Cool thanks! Pro's and con's of doing it client side, but I think the progress will be helpful regardless and the tradeoffs aren't too important for the proof of concept and we can always supply it from the DB ultimately. Where did you get your source of districts by the way? We'll probably have to update that for the 2020 census.

**Comment by rkaufman13** [comment]
I assumed that eventually y'all would want to store the data on the back
end but I figured it should be trivial to wire that up later. Data is from
ballotpedia which I find more accessible than official Gov sites, and
almost always correct. If you see a typo /missing district speak up of
course!

**Comment by rkaufman13** [comment]
(Also yes, data is current as of Aug 2021 so the numbers should be correct
as to numbers of districts, I believe)

File: src/components/AddOrder.js [hunk 1]
```diff
@@ -1,14 +1,15 @@
 import React, { useState } from "react";
 import OrderDataService from "../services/OrderService";
  
-const STATES = require('./states.json');
+//const STATES = require('./states.json');
```
Comment (ethanstrominger) [inline src/components/AddOrder.js:4]: Remove comment

**Review by ethanstrominger — APPROVED** [review]
Woo hoo, great to see a pull request so soon after joining.

**Pull request merged**